components/TxForm.tsx

```diff
@@ -39,13 +22,6 @@ export const TxForm = () => {
     "function transfer(address,uint256) returns (bool)",
     "function approve(address,uint256) returns (bool)",
   ];
-  console.log(
-    getRailgunSmartWalletContractForNetwork(NetworkName.EthereumGoerli).address
-  );
-  const { isConnected, address } = useAccount();
-  const { data: signer } = useSigner();
-  const provider = useProvider();
-  const { getShieldPrivateKey } = useShieldPrivateKey();
   const { config } = usePrepareContractWrite({
     address: "0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6",
     abi: erc20ABI,
```

Comment:
Above, the `abi` variable is declared but never used

Reply:
fixed!